lib/common/database.dart

```diff
@@ -25,14 +28,19 @@ class DatabaseService {
   DatabaseService.createTest({required String uid, required firestoreObject}) {
     userid = uid;
     users = firestoreObject.collection('users');
+    fs = firestoreObject;
   }
 
+  get firestoreObject => fs;
+
   /// Assign UID. This must be ran before any other database function is called else it will crash
   /// takes the string ID
   initUID(String uid) {
     userid = uid;
   }
 
+////////////////////////////////////////////////////
```

Comment:
Can you add a description of what this is for?

Reply:
Just a visual separator :-)